README.md

```diff
@@ -24,7 +24,9 @@
 
 ## Try It Yourself
 
-More installation and usage details can be find in our [documentation](http://www.scalabel.ai/doc). It also includes Windows setup.
+Below is a quick way to install dependencies and launch the Scalabel server. After launching the server, you can directly jump to [how to use the tool](#using-scalabel). If you want to know more about the installation process, please check [additional tips](#installation-tips).
+
+**Note**: You only need to do either Step 3 or 4, but not both.
 
 1. Check out the code
 
```

Comment:
Is there a need for the first part of this command? It works better form me to do:
git clone https://github.com/scalabel/scalabel